File: BFS/debug.java
public class debug {
    static int sum(int num) {
        int total = 0;
        for (int i = 0; i < num; i++) {
          total += i;
          printTotal(total);
        }
        return total;
      }
      
      static void printTotal(int num) {
       System.out.println(num);
      }
      public static void main(String[] args){
        int result = sum(2);
        System.out.println(result);
      }
      
}

File: BFS/debug.js
function sum(num) {
  let total = 0;
  for (let i = 0; i < num; i++) {
    total += i;
    printTotal(total);
  }
  return total;
}

function printTotal(num) {
  console.log("ddd");
}

const result = sum(2);
console.log(result);
